sympy/stats/joint_rv_types.py

```diff
@@ -251,12 +254,25 @@ def NormalGamma(syms, mu, lamda, alpha, beta):
     Parameters
     ==========
 
-    syms: list/tuple/set of two symbols for identifying each component
+    syms: A symbol/str for identifying the random variable.
```

Comment:
Was this just wrong before? If it's really only supposed to be one symbol the parameter should be called `sym`.

Reply:
I think it was wrong before as when going into function definitions of ```multivariate_rv```, it is clearly mentioned that it takes only **a** symbol, not multiple ones. I will update ```syms``` to ```sym```.